lib/filterable/filter.rb

```diff
@@ -53,6 +45,18 @@ def type_validator
 
     private
 
+    def apply_scope_filter(collection, value, active_sorts_hash, params)
+      if value.present? && scope_params.empty?
+        collection.public_send(internal_name, *parameter(value))
```

Comment:
The `*parameter(value)` breaks a bunch of procore filters & tests I believe

Reply:
I suspect those scopes on the Procore side need to change. I am not sure how we can support both behaviors.